src/DwitTech.AccountService.Core/Interfaces/IActivationService.cs

```diff
@@ -8,6 +9,7 @@ namespace DwitTech.AccountService.Core.Interfaces
 {
     public interface IActivationService
     {
-        bool SendActivationEmail(string fromEmail, string toEmail, string templateName, string RecipientName, string subject, string cc, string bcc);
+        
+        Task<bool> ActivateUser(string activationCode, string fromEmail, string toEmail, string templateName, string subject = "Account Details", string cc = "", string bcc = "");
```

Comment:
There shouldn't be hardcoded values in your interface

Reply:
> There shouldn't be hardcoded values in your interface

Resolved

Reply:
Why do you think there should not be hardcoded values in the interface?